ui/src/app/components/help-sidebar.tsx

```diff
@@ -475,10 +476,9 @@ export const HelpSidebar = fp.flow(withCurrentWorkspace(), withUserProfile())(
             <div style={styles.footer}>
               <h3 style={{...styles.sectionTitle, marginTop: 0}}>Not finding what you're looking for?</h3>
               <p style={styles.contentItem}>
-                Visit the <StyledAnchorTag href='https://www.researchallofus.org/frequently-asked-questions/'
-                                           target='_blank' onClick={() => this.analyticsEvent('FAQ')}> FAQs here
-                </StyledAnchorTag> or <span style={styles.link} onClick={() => this.openContactWidget()}> contact
-                us</span>.
+                Visit our <StyledAnchorTag href={environment.zendeskHelpCenterUrl}
+                                           target='_blank' onClick={() => this.analyticsEvent('UserSupport')}> User Support
+                </StyledAnchorTag> page or <span style={styles.link} onClick={() => this.openContactWidget()}> contact us</span>.
```

Comment:
I realize you're matching the text in the Jira ticket here, but shouldn't `page` be part of the anchor text?

Reply:
So I checked with Helen, and she pointed me to this [NIH style guide](https://docs.google.com/document/d/1q3qgH8A3h9-al7bFxVcTq7-L0ZKqNiUiSPE35HoD4uc/edit) that specifies not to include "page" in hyperlinks.

Reply:
Ah, great. thanks for following up on that.